playbooks/optimize_kernel_scheduler_for_vmware_guests.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: Cloud | VMware | Enable noop kernel param for RHEL VMs on VMware
```

Comment:
@curt-matthews out of matter of consistency i try to make the play name match the file name. So maybe change this to `Cloud | Optimize Kernal Scheduler` and then also rename the file to `optimize_kernel_scheduler.yml` that way later if we find a kernel scheduler optimization for AWS, or Azure, or Google, or RHV, or etc, we can just add them to this one playbook.